der/tests/derive.rs

```diff
@@ -218,8 +218,7 @@ mod sequence {
     pub struct SubjectPublicKeyInfo<'a> {
         pub algorithm: AlgorithmIdentifier<'a>,
 
-        #[asn1(type = "BIT STRING")]
-        pub subject_public_key: &'a [u8],
+        pub subject_public_key: BitString<'a>,
```

Comment:
Why did you change this?

Reply:
Because it made the default stuff messy for not particular value to my eye. We could add it back with cost of some additional verbosity in the default bits. If we do restore, I'd prefer to wait until context specific proc macro support lands (or address in tandem with that).

Reply:
I'd really prefer to retain the functionality, although I'd be happy with fixing it in a follow-up PR after merging this one